subcommand/mesh-init/command.go

```diff
@@ -63,31 +60,68 @@ func (c *Command) Run(args []string) int {
 }
 
 func (c *Command) realRun() error {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
 	taskMeta, err := awsutil.ECSTaskMetadata()
 	if err != nil {
 		return err
 	}
 
 	cfg := api.DefaultConfig()
 
+	consulClient, err := api.NewClient(cfg)
+	if err != nil {
+		return fmt.Errorf("constructing consul client: %s", err)
+	}
+
+	serverConnMgrCfg, err := c.config.ConsulServerConnMgrConfig(taskMeta)
+	if err != nil {
+		return fmt.Errorf("constructing server connection manager config: %s", err)
+	}
+	serverConnMgrCfg.ServerWatchDisabled = false
```

Comment:
We'll need a config option for server watch disabled. Users will need to disable the server watch whenever their Consul servers are behind a load balancer (per [the docs](https://developer.hashicorp.com/consul/docs/connect/dataplane/consul-dataplane#server-watch-disabled))